canonical-web-and-design/snapcraft.io
# Update some dependencies

## Done

- Update some dependencies

## Issue / Card

Fixes #

## QA

- Pull the branch
- Run the site using the dotrun snap with `$ dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8004/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See everything works as expected

## Screenshots

[if relevant, include a screenshot]
